contributors.json

```
@@ -114,5 +114,8 @@
     {
         "name": "Wumpacraft",
         "links": ["https://twitter.com/wumpacraft"]
+    },
+    {
+        "name":"foodiebonus"
```

```suggestion
        "name":"foodiebonus",
        "links": []
```
I think the field needs to exist. CC @misode